ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -223,20 +223,29 @@
   :subject "Email Subscription Notification"
   :body [{:type "text/html"
    :content (markdown/md-to-html-string (str
-    "\nYou have subscribed to receive notifications when the following query is updated:\n\n"
-    (get-in subscription [:metadata :Query])
-    "\n\nSince this query was last run at"
-    (get-in subscription [:metadata :updated_since])
+    "\nYou have subscribed to receive notifications when data is added to the following query:\n\n"
+    "`" (get-in subscription [:metadata "CollectionConceptId"]) "`\n\n"
```

Comment:
Should move the string extractions up into a `let` block for easier curation

Reply:
done, thanks